locale/de_DE/locale.xml

```diff
@@ -16,20 +16,20 @@
 	<message key="monograph.audience">Zielgruppe</message>
 	<message key="monograph.coverImage">Titelbild</message>
 	<message key="monograph.currentCoverImage">Aktuelles Bild</message>
-	<message key="monograph.currentCoverImageReload">Speichern Sie Ihre Änderungen, um das neue Titelbild zu sehen.</message>
+	<message key="monograph.currentCoverImageReload">Speichern Sie Ihre Änderungen, um das neue Titelbild angezeigt zu bekommen.</message>
 	<message key="monograph.audience.rangeQualifier">Grundlage der Eingrenzung der Zielgruppe</message>
 	<message key="monograph.audience.rangeFrom">Bildungsstand (von)</message>
 	<message key="monograph.audience.rangeTo">Bildungsstand (bis)</message>
 	<message key="monograph.audience.rangeExact">Bildungsstand (genau)</message>
 	<message key="monograph.languages">Sprachen (Englisch, Französisch, Spanisch)</message>
 	<message key="monograph.publicationFormats">Publikationsformate</message>
 	<message key="monograph.publicationFormat">Format</message>
-	<message key="monograph.publicationFormatDetails">Details über das verfügbare Publikationsformat {$format}</message>
-	<message key="monograph.miscellaneousDetails">Details über diese Monographie</message>
+	<message key="monograph.publicationFormatDetails">Genauere Angaben zum verfügbaren Publikationsformat {$format}</message>
+	<message key="monograph.miscellaneousDetails">Angaben zur Monographie</message>
 	<message key="monograph.carousel.publicationFormats">Formate:</message>
 	<message key="monograph.type">Art der Einreichung</message>
 	<message key="submission.pageProofs">Korrekturfahnen</message>
-	<message key="monograph.proofReadingDescription"><![CDATA[Der Layouter/die Layouterin lädt hier die für die Publikation vorbereiteten produktionsreifen Dateien hoch. Verwenden Sie <em>+Zuweisen</em>, um jene Autor/innen und anderen Beteiligten zu bestimmen, die diese Dateien Korrektur lesen und ihre Zustimmung zur Veröffentlichung geben sollen. (Bitte die korrigierten Dateien mit hochladen.)]]></message>
+	<message key="monograph.proofReadingDescription"><![CDATA[Der Layouter lädt hier die für die Publikation vorbereiteten druckfertigen Dateien hoch. Verwenden Sie <em>+Zuweisen</em>, um jene Autoren und anderen Beteiligten zu bestimmen, die diese Dateien Korrektur lesen und ihre Zustimmung zur Veröffentlichung geben sollen. (Bitte die korrigierten Dateien mit hochladen.)]]></message>
```

Comment:
This would break consistency with the rest of the translation and its gender-neutral approach.